app/grandchallenge/datatables/views.py

```diff
@@ -44,7 +45,7 @@ def get(self, request, *args, **kwargs):
         response = super().get(request, *args, **kwargs)
         if request.META.get("HTTP_X_REQUESTED_WITH") == "XMLHttpRequest":
             start = int(request.GET.get("start", 0))
-            page_size = int(request.GET.get("length"))
+            page_size = self.page_size or int(request.GET.get("length"))
```

Comment:
Does this really work with Datatables JS? If it is requesting 50 objects and you only send back 10, is the pagination correct (eg, total number of pages and objects that appear on subsequent pages)? I would imagine that you would set the pagination options client side, and then set a maximum server side, raising a 400 bad request error if the clients selected pagination is over the server side limit.